### PR TITLE
Disabling Atlas

### PR DESCRIPTION
**Pull request by lgalabru**

### CODE REVIEW

**Review by jcnelson — COMMENTED** [review]
I think you should also disable `handle_getattachment` just to be safe.

File: src/net/rpc.rs [hunk 1]
```diff
@@ -589,90 +589,96 @@ impl ConversationHttp {
...
-                let msg = format!("Unable to read Atlas DB");
-                warn!("{}", msg);
-                let response = HttpResponseType::ServerError(response_metadata, msg);
-                response.send(http, fd)?;
-                return Err(net_error::DBError(e));
-            }
-        }
+        let msg = format!("Atlas disabled");
+        warn!("{}", msg);
+        let response = HttpResponseType::ServerError(response_metadata, msg);
```
Comment (jcnelson) [inline src/net/rpc.rs:594]: Can you make this 404 instead?

File: src/net/rpc.rs [hunk 2]
```diff
@@ -589,90 +589,96 @@ impl ConversationHttp {
...
+        response.send(http, fd)?;
+        return Err(net_error::DBError(e));
```
Comment (jcnelson) [inline src/net/rpc.rs:596]: What's `e` here?

Comment (lgalabru) [reply src/net/rpc.rs]: Address with latest

File: src/net/rpc.rs (hunk 1 above)
Comment (lgalabru) [reply src/net/rpc.rs]: Address with latest

**Review by jcnelson — APPROVED** [review]

**Review by kantai — APPROVED** [review]

**Pull request merged**